### DISCUSSION TITLE
Request Feature

### DISCUSSION THREAD

**Discussion by hirockstar** [spencerwooo/onedrive-vercel-index#281] [Ideas]
Add copy link button in mobile mode too.

**Comment by spencerwooo** [comment]
![image](https://user-images.githubusercontent.com/32114380/149656433-64ad9996-6691-4150-a1dc-d4e9cbfd297c.jpeg)

**Reply by hirockstar** [reply]
[](url)
![Screenshot 2022-01-16 181619](https://user-images.githubusercontent.com/45270927/149660494-da0120f0-f17d-42c9-909d-66d7e01ea974.png)
talking about this bro.

**Reply by spencerwooo** [reply]
nope, we don't have space on mobile, and I don't want the file listing to be able to be scrolled horizontally

**Reply by hirockstar** [reply]
okey got it.
BTW what about search button is there any plans for that?
